src/parser/bison_parser.y

```diff
@@ -517,7 +529,97 @@ import_export_options : import_export_options ',' FORMAT file_type {
 | ENCODING STRING {
   $$ = new ImportExportOptions{};
   $$->encoding = $2;
-};
+}
+| import_export_options ',' csv_option {
```

Comment:
Given the current approach: can we use encoding and csv options?

Reply:
Yes, that should be parsed properly. I assume however, that for CSV files, we shouldn't have an encoding?

Reply:
No, good that it works. The parser syntax just always confuses me. I need to take a pass once again.

Reply:
Do we need to handle the code between 533 and 604 like that? A lot of repetition and I don't even know why. :D